projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.php

```diff
@@ -275,3 +275,20 @@ function wpcom_custom_wpcom_admin_bar_class( $wp_admin_bar_class ) {
 	return '\Automattic\Jetpack\Jetpack_Mu_Wpcom\WPCOM_Admin_Bar';
 }
 add_filter( 'wp_admin_bar_class', 'wpcom_custom_wpcom_admin_bar_class' );
+
+/**
+ * Changes the edit site menu to point to the top-level site editor.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar The WP_Admin_Bar core object.
+ */
+function wpcom_edit_site_menu_override( $wp_admin_bar ) {
+	if ( $wp_admin_bar->get_node( 'site-editor' ) ) {
+		$args = array(
+			'id'   => 'site-editor',
+			'href' => admin_url( 'site-editor.php' ),
+		);
+
+		$wp_admin_bar->add_node( $args );
+	}
+}
+add_action( 'admin_bar_menu', 'wpcom_edit_site_menu_override', 9999 );
```

Comment:
I guess it should run after after the original menu item is created and after any other, possible `admin_bar_menu` hooks, but I was just curious where there something special about `9999`? 

The [origin menu item priority in Core](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-admin-bar.php#L660) is `40`.

Reply:
nothing special. Saw 9999 being used elsewhere in the file and went with that to be safe with so many `admin_bar_menu` hooks groked in wpcom. 

> The [origin menu item priority in Core](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-admin-bar.php#L660) is 40

Thanks, `41` works in my testing. Went ahead and updated to that.